googlemaps/places.py

```diff
@@ -0,0 +1,203 @@
+#
+# Copyright 2015 Google Inc. All rights reserved.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+"""Performs requests to the Google Maps Places API."""
+
+from googlemaps import convert
+
+
+def places(client, query, location=None, radius=None, language=None,
+           min_price=None, max_price=None, open_now=False, types=None,
+           page_token=None):
+    """
+    Performs text search for places.
```

Comment:
That description was to distinguish it from the other search methods that were previously there, but those have since been removed based off some offline feedback from @domesticmouse - I've updated the description to better match it here: a58c0ed18fd6d1105aaafd1e73bd46133297b085